### ISSUE TITLE
Maybe create an 'update page' seprately

### ISSUE THREAD

**Issue by NripPrajapat** [ImranR98/Obtainium#2001]
A page next to 'Apps' and before 'Add App', where only Update Avaible apps are listed, and could choose among those apps, which one to update.
Because in categorised apps section, will have to scan through all the apps or list to find which app has an update.
And if I press, Download button it only provide an option which allows user to only update 'all the available', can't select among them at all.

**Comment by ImranR98** [comment]
I don't think it makes sense to add a whole extra page. Instead we could have a toggle that, when enabled, groups all in a new "updates" category that is pinned to the top (only visible when there are updates).

**Comment by DarkCrypt** [comment]
> Because in categorised apps section, will have to scan through all the apps or list to find which app has an update.

I agree this is slightly annoying. ImranR98's suggested solution should suffice.

> And if I press, Download button it only provide an option which allows user to only update 'all the available', can't select among them at all.

Incorrect, the ability to select which apps to update is already implemented. Long-press the 1st app to enter multi-select mode (from Wiki [Apps Page](https://wiki.obtainium.imranr.dev/ui_overview/#apps-page)) and then tap on whatever other apps you want to update.

**Comment by NripPrajapat** [comment]
> I don't think it makes sense to add a whole extra page. Instead we could have a toggle that, when enabled, groups all in a new "updates" category that is pinned to the top (only visible when there are updates).

Yes, that could work too, I'll be the same, when one doesn't make categories. That will solve it too.
Thmain point for this whole was to ease up the updating process, it doesn't matter how it is done, either be it a page or pinned category.
Cheers.
Thanks for your time and consideration.

**Comment by NripPrajapat** [comment]
> > Because in categorised apps section, will have to scan through all the apps or list to find which app has an update.
> 
> I agree this is slightly annoying. ImranR98's suggested solution should suffice.
> 
> > And if I press, Download button it only provide an option which allows user to only update 'all the available', can't select among them at all.
> 
> Incorrect, the ability to select which apps to update is already implemented. Long-press the 1st app to enter multi-select mode (from Wiki [Apps Page](https://wiki.obtainium.imranr.dev/ui_overview/#apps-page)) and then tap on whatever other apps you want to update.

First point I agree with.
But with second point I don't, because in that process, still I have to find and choose all the listed apps, it's irritating when one have like more than 300+ apps to watch for.
